src/tool_help.c

```diff
@@ -272,6 +272,7 @@ static const char *const helptext[] = {
   "     --tlspassword STRING  TLS password",
   "     --tlsauthtype STRING  TLS authentication type (default: SRP)",
   "     --unix-socket FILE    Connect through this Unix domain socket",
+  "     --abstract-unix-socket PATH Connect to an abstract Unix domain socket",
```

Comment:
For consistency with `--unix-socket`, maybe change FILE to PATH (or the other way round?)